Makefile

```diff
@@ -41,7 +41,7 @@ ifndef SR_XML
 endif
 
 build:
-	jbuilder build \
+	jbuilder build --dev \
```

Comment:
`--dev` should be enabled only in dev builds, not on the release ones. We can't control which flags it can enable for the moment

Reply:
the default goal is `sdk_all` which does not call the `build` any more; it calls each one of the language targets and these call `jbuilder build` directly without `--dev`.

Reply:
I see, you are right. Then it's good to have the flag on there